replset-consistency/README.md

```diff
@@ -104,13 +109,20 @@ The overall high level process is:
     1. Optionally, re-run `dbCheck` to confirm consistency
     1. Resume writes to the collection.
 
-## Validate
+## Checking for Local Inconsistencies
 
-For each node, run `validate` on all collections on all nodes. `validate.js` runs validate on every collection of a given node.
+For each node, run `validate` on all collections on all nodes. [`validate.js`](https://github.com/mongodb/support-tools/blob/master/replset-consistency/validate.js) runs validate on every collection of a given node.
 
-On each node with validation issues, any “missing index entries” for the `_id` index must be fixed using [`reIndex`]|(https://docs.mongodb.com/manual/reference/method/db.collection.reIndex/) prior to running `dbCheck`. Other index inconsistencies, including extra entries in the `_id` index, do not need to be addressed prior to `dbCheck` (but will not necessarily be resolved by this remediation).
+On each node with validation issues, any “missing index entries” for the `_id` index must be fixed using [`reIndex`](https://docs.mongodb.com/manual/reference/method/db.collection.reIndex/) prior to running `dbCheck`. Other index inconsistencies, including extra entries in the `_id` index, do not need to be addressed prior to `dbCheck` (but will not necessarily be resolved by this remediation).
 
-## Check and Remediate
+If you are completing this process to check for possible replica set inconsistency that requires running dbcheck if validation fails (such as WT-10461), proceed with the steps in [Checking for Replica Set Inconsistencies](#Checking-for-Replica-Set-Inconsistencies)**.
+
+If you are not checking for possible replica set inconsistency, any nodes that fail validation should be initial-synced from a node that passes validation. No further steps are required.
```

Comment:
Should we point at step 5 here and use step 5 to better detail remediation with initial sync?